Oracle.md

```diff
@@ -204,6 +204,89 @@ The algorithm `Sign(sk, message, tag)` is defined as:
 * Let `m` = H(`message`)
 * Return `BIP340_sign(sk, m)`
 
+## JSON Serialization
+
+In addition to the previously define TLVs, we also provide recommendation on JSON serialization of oracle messages, in order to facilitate interoperability between oracle and clients using this format.
+
+### Announcements
+
+The following example describe JSON serialization of oracle announcements:
+
+```json
+{
+   "announcementSignature": "464410ffd8be04249a6ea6a646fd07a05d8afbd8f6a96a6a6339ed66f799383b464410ffd8be04249a6ea6a646fd07a05d8afbd8f6a96a6a6339ed66f799383b",
+   "publicKey": "ce4b7ad2b45de01f0897aa716f67b4c2f596e54506431e693f898712fe7e9bf3",
+   "event":{
+      "nonces":[
+         "5ebf5cdbcafe17b7b419b728e68cde1d7035618bab4e4732971149b4a1c7ecd8",
+      ],
+      "maturity":"2021-01-14T07:20:00Z",
+      "descriptor":{
+         ...
+      },
+      "eventId": "btcusd1610608800"
+   }
+}
+```
+
+where:
+`anouncementSignature` is a hex string of length 128 representing a BIP340 Schnorr signature,
```

Comment:
of what? Presumably a serialization of this JSON string without the signature field or something like that?